server/routerlicious/packages/routerlicious-base/src/alfred/runnerFactory.ts

```diff
@@ -257,15 +257,6 @@ export class AlfredResourcesFactory implements core.IResourcesFactory<AlfredReso
 			config.get("checkpoints:checkpointsTTLInSeconds") ?? defaultTTLInSeconds;
 		await checkpointsCollection.createTTLIndex({ _ts: 1 }, checkpointsTTLSeconds);
 
-		// Foreman agent uploader does not run locally.
-		// TODO: Make agent uploader run locally.
-		const foremanConfig = config.get("foreman");
-		const taskMessageSender = services.createMessageSender(
-			config.get("rabbitmq"),
-			foremanConfig,
-		);
-		await taskMessageSender.initialize();
-
```

Comment:
These were the ones [removed before](https://github.com/microsoft/FluidFramework/pull/15129) and then [re-added by mistake](https://github.com/microsoft/FluidFramework/pull/14627).

This seems to be the only use of `createMessageSender()` and thus of `RabbitmqTaskSender` inside it, which tracks with something @pradeepvairamani told me about RabbitMq not being in use anymore. But I see recent changes in `RabbitmqReceiver` which leave me unsure about also cleaning up all references (and code?) related to RabbitMq. @znewton the recent change there [was yours](https://github.com/microsoft/FluidFramework/pull/16368). Was that more of a "go through all floating promises in the repo and fix them" or does it actually mean that the class is still in use? The previous meaningful change to it is from 2021.

Reply:
I think RabbitMQ is not in use indeed, and @znewton 's changes were like you said: "go through all floating promises and fix them". So I think it is safe to remove this.